doc/tools.md

```diff
@@ -112,3 +114,14 @@ Removes the zip files in the <tt>tmp</tt> folder that's created when you package
 
     $ zat clean
 
+### Create
+
+Packages your app directory into a zipfile, then uploads it into a Zendesk account. Also stores the new app's ID and other metadata in a zatfile (by default, this is `.zat`).
```

Comment:
zipfile -> zip file (2 words)
zatfile -> zat file (2 words if not referring to the command-line option `zatfile`)